aspnetcore/tutorials/signalr.md

```diff
@@ -69,6 +69,7 @@ At the end, you'll have a working chat app:
    ```dotnetcli
    dotnet new webapp -o SignalRChat
    code -r SignalRChat
+   cd SignalRChat
```

Comment:
Well technically if you follow the instructions you'll be running the commands in the VSCode terminal, which should be in the correct directory.

However, I'm fine with this change if you move this up one line, and then change `code -r SignalRChat` to `code -r .`

Reply:
I have just done exactly as you said — moved cd command one line up.